account_vat_period_end_statement/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,45 @@
+**Italiano**
+
+    Per fare la liquidazione IVA, aprire Fatturazione > Contabilità > Liquidazioni IVA, il menù è visibile solo quando è abilitato il gruppo 'Mostrare funzionalità contabili complete'.
+    Selezionare un registro che conterrà le registrazioni contabili della liquidazione.
+    Il campo 'Conto IVA Erario' contiene il conto dove verrà effettuata la registrazione della liquidazione IVA.
+
+    L'oggetto 'Liquidazione IVA' permette di specificare ogni importo e il conto utilizzato dalla liquidazione.
+    Di norma, gli importi di debito e credito delle tasse vengono caricati automaticamente dai periodi selezionati
+    (vedere Configurazione per generare correttamente i periodi).
+    I debiti e crediti precedenti vengono caricati dalle liquidazioni IVA precedenti, in base allo stato del loro pagamento.
+
+    Per creare la registrazione contabile, cliccare sul bottone 'Crea movimento', dentro il tab 'Conti'.
+    Se i termini di pagamento sono impostati viene scritta anche la scadenza (o le scadenze).
+
+    Il tab 'Erario' contiene informazioni sui pagamenti,
+    qui si possono visualizzare i risultati della liquidazione ('Importo IVA erario')
+    e l'importo residuo da pagare ('Importo a saldo').
+    La liquidazione può essere pagata come qualunque altro debito, con la riconciliazione delle registrazioni contabili.
+
+    È inoltre possibile stampare la liquidazione IVA cliccando su Stampa > Stampa liquidazione IVA.
```

Comment:
s/cliccando su/con un clic su (o facendo clic su)

Reply:
Anche qui, se non è un problema manterrei cliccare.
È solo un fatto di gusti o cliccare è proprio sbagliato?

Reply:
Storicamente si usa "fare clic", nella documentazione si tende a essere formali e impersonali, "cliccare" è più colloquiale.

Inoltre, cosa non trascurabile, risolve in modo elegante e senza ambiguità la traduzione di "double click" → "fare doppio clic".
Se usi "cliccare" la traduzione sarebbe "cliccare due volte", un po' ambiguo perché non trasmette il senso di due clic ravvicinati.

Per le traduzioni direi che è meglio mantenere "fare clic", per quanto riguarda la documentazione potremmo anche decidere diversamente, ma resterebbe sempre l'eventuale problema dell'ambiguità.

Si potrebbe anche decidere di usare "cliccare" solo per nella documentazione dei README, ma a quel punto per uniformità dovrebbero essere adattati i README di tutti gli altri moduli.

Reply:
@primes2h ok, a me sembra più normale cliccare ma forse perché sono un'altra generazione; capisco che ad altri possa sembrare più naturale _fare clic_

Reply:
Il fatto che "cliccare" sembri più normale è normale :stuck_out_tongue: e non dipende da generazioni diverse.
Dipende solo dal fatto che si usa molto nel parlato e nello scritto informale e diretto.

Se invece parliamo di documentazione tecnica di un certo livello è normale l'uso di "fare clic".

Riporto giusto qualche esempio di documentazione di aziende conosciute:

https://support.office.com/it-it/article/documenti-attendibili-cf872bd8-47ec-4c02-baa5-1fdba1a11b53
https://www.ibm.com/support/knowledgecenter/it/SSWLGF_8.5.0/com.ibm.sr.webui.doc
/html/load_documents_initial_screen.html
https://helpx.adobe.com/it/acrobat/using/securing-pdfs-certificates.html
https://support.avast.com/it-it/article/Use-Antivirus-Sensitive-Data-Shield
https://support.hp.com/it-it/document/c05630674

Se noti, oltre a "fare clic", rispettano anche tutte le classiche linee guide della buona documentazione, come ad esempio l'uso dell'impersonale.